client-v2/src/components/FrontsEdit/CollectionContainer.js

```diff
@@ -6,55 +6,56 @@ import { bindActionCreators } from 'redux';
 import getFrontCollection from '../../actions/Collection';
 import getArticlesForCollection from '../../actions/Articles';
 import CollectionDetail from './CollectionDetail';
-
+import { collectionSelector } from '../../selectors/collectionSelectors';
+import { collectionArticlesSelector } from '../../selectors/collectionArticleSelectors';
 import type { ConfigCollectionDetailWithId } from '../../types/FrontsConfig';
 import type { Collection } from '../../types/Collection';
-import type { CapiArticle } from '../../types/Capi';
 import type { State } from '../../types/State';
+import type { CapiArticle } from '../../types/Capi';
 
 type Props = {
-  collection: ConfigCollectionDetailWithId
+  collectionConfig: ConfigCollectionDetailWithId,
+  browsingStage: string // eslint-disable-line react/no-unused-prop-types
```

Comment:
If this isn't used, does it need to be here?

Reply:
It is used because we are passing it to the article selector but as this happens outside the component itself, eslint gets confused and thinks it's an error.